render.yaml

```diff
@@ -0,0 +1,23 @@
+databases:
+  - name: postgres
+    ipAllowList: []
+    plan: free # optional (defaults to starter)
+
+services:
+  - type: web
+    name: rails
+    env: ruby
+    buildCommand: "./bin/render-build.sh"
+    startCommand: "bundle exec rails s"
+    envVars:
+      - key: DATABASE_URL
+        fromDatabase:
+          name: postgres
+          property: connectionString
+      # - key: RAILS_MASTER_KEY
+      #   sync: false
+  - type: redis
```

Comment:
It's interesting that redis is a "service" while Postgres is a "database"

Reply:
Based on my understanding, redis is firstly an in-memory storage used for caching purposes, and is not **technically** intended to be for used where persistence is critical (in practice I believe this isn't absolutely true due to its default 1-second flush to file settings, but I can't say I've tested anything in this area yet). Without looking more into it, this may be the reason why.

Reply:
Ahhh that makes sense. Definitely seems like a render-specific thing.